src/components/Comments.jsx

```diff
@@ -1,37 +1,104 @@
 import { useState, useEffect } from "react";
-import { getComments } from "../utils";
-import '../Comments.css'
-
-const Comments = ({article_id}) => {
-      const [comments, setComments] = useState([])
-      const [isLoading, setIsLoading] = useState(true)
-
-    useEffect(() => {
-        const fetchComments = async () => {
-            const response = await getComments(article_id)
-            setComments(response.comments)
-            setIsLoading(false)
-        }
-        fetchComments()
-    }, [])
-
-    
-    if (isLoading === true){
-        return <p>Loading...</p>
+import { getComments, postComment } from "../utils";
+import "../Comments.css";
+
+const Comments = ({ article_id, userId }) => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [newComment, setNewComment] = useState("");
+  const [inputError, setInputError] = useState("");
+  const [submitStatus, setSubmitStatus] = useState("");
+  const [formDisabled, setFormDisabled] = useState(false);
+
+  useEffect(() => {
+    const fetchComments = async () => {
+      const response = await getComments(article_id);
+      setComments(response.comments);
+      setIsLoading(false);
+    };
+    fetchComments();
+  }, []);
+
+  const handleChange = (e) => {
+    setNewComment(e.target.value);
+  };
+
+  const addComment = async () => {
+    const newPost = { username: userId, body: newComment };
+    const response = await postComment(article_id, newPost);
+    console.log(response)
```

Comment:
Please remove console.logs - remember your PRs should be demonstrating code that could be considered production ready